preprocess_text.py

```diff
@@ -4,13 +4,22 @@
 import numpy as np
 import pandas as pd
 
-def tokenize_a_doc(s):
+from bs4 import BeautifulSoup
+from HTMLParser import HTMLParser
+
+def tokenize_a_doc(s, hook_func=None):
+    if hook_func is not None:
+        s = hook_func(s)
+    return word_tokenize(' '.join(s.split(" ")).lower())
+
+def parse_html(s):
     try:
```

Comment:
I think this try/catch needs to be in the tokenize_a_doc method, not the html-parser?